src/Game.java

```diff
@@ -0,0 +1,64 @@
+import java.util.*;
+
+public class Game {
+    private List<String> words = new ArrayList<>();
+    Random random = new Random();
+    WordsDatabase database = new WordsDatabase(words, random);
+    public final int exit = 0;
+    public final int play = 1;
+    private int opition = 1;
+    Scanner sc = new Scanner(System.in);
+
+    public void hangman() {
+        try {
+            while (opition != exit) {
+
+
+                switch (opition) {
+                    case exit:
+                        System.out.println("Koniec gry.");
+                        sc.close();
+                        break;
+                    case play:
+                        game();
+                        break;
+                    default:
+                        System.out.println("Nie ma takiej opcji.");
+                }
+                System.out.println("Dostępne opcję:");
```

Comment:
Change the way that this method works. In your code after starting the app, game starts immediately after. What I start the app and want to exit without playing a game?

Reply:
I think this same, but after i saw how the game is printed in task example i change it as the same, but i will change it.

Reply:
According to your question, if someone want to ends game before play, why he starts game ?